service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/NotifyGatekeeperController.java

```diff
@@ -28,6 +28,8 @@
     private final ValidateGroupService validateGroupService;
     private final ApplicationEventPublisher applicationEventPublisher;
 
+    //TODO: can we validate a hearing has been added at this point? Saves some nasty exceptions in the case of
+    // no hearing being present when populating standard directions FPLA-1516
```

Comment:
Do we want to keep that comment if we've got ticket raised for it?

Reply:
yes, I am inclined to leave them. Comments are good imo